### DISCUSSION TITLE
MS Minecraft and solitaire

### DISCUSSION THREAD

**Discussion by mgkai** [Klocman/Bulk-Crap-Uninstaller#366] [Q&A] [answered]
Hi, using BC 5.2 wanted to remove MS Minecraft and solitaire (scattered mostly inside C:\Program Files\WindowsApps).
Take 1 GB of disk space ... But not listed in BC 5.2. Any pointers? Thank you!

**Comment by somedudelikesit** [comment, marked as answer]
try take ownership and delete
here's the link to registry editor
it really works for deleting outside owned stuff
https://www.majorgeeks.com/mg/getmirror/take_full_ownership_of_files_folders_registry_hack,1.html
